tox.ini

```diff
@@ -32,17 +32,28 @@ exclude = .tox,.cache,.pytest_cache
 [testenv:integrationtests]
 basepython = python3.8
 deps =
-	-r requirements/base.txt
+	-rrequirements/dev.txt
+passenv = LANG TERM LANGUAGE LC_ALL LD_LIBRARY_PATH BRANCH INTEGRATION_TESTS_REPO_URL TARGET_REPO
 setenv =
     API_CLIENT_RUN_ON_REDIS = true
+	INT_TESTS_REPO_SSH_URL = git@github.com:gridsingularity/gsy-backend-integration-tests.git
```

Comment:
Here we don't necessarily need to use the SSH version, right? I have the feeling that this variable could be renamed to `INTEGRATION_TESTS_REPO_URL` (or whatever you like). Then the CI can fill its value with a SSH URL, while locally we can decide to also use HTTPS. What do you think?